debian/syslog-ng-core.install

```diff
@@ -57,7 +57,6 @@ usr/share/syslog-ng/xsd/*
 [!kfreebsd-any] debian/syslog-ng.systemd =>  lib/systemd/system/syslog-ng.service
 
 debian/syslog-ng.conf                   /etc/syslog-ng/
-debian/scl.conf                         /etc/syslog-ng/
```

Comment:
Just FYI, this was the commit message of this change:

    debian: Install a custom scl.conf
    
    Upstream uses an scl.conf that includes the various snippets one-by-one,
    explicitly. This does not allow us to split them out of syslog-ng-core
    into various other packages where they may be more fitting. To fix this,
    ship our own scl.conf, that uses a wildcard include.
    
    (See balabit/syslog-ng#296 for the upstream pull request.)

Reply:
I think this was because we lacked support for wildcarded directories (?), it has been using wildcards since 2014 upstream too:

```
Author: Balazs Scheidler <balazs.scheidler@balabit.com>  2014-11-17 17:26:37
Committer: Balazs Scheidler <balazs.scheidler@balabit.com>  2014-11-22 10:11:29
Parent: c3ab1e83c2577c0abeba96f9d544abfbfc8d5c92 (scl: add $(format-cim) template function)
Child:  d677e3119a2b2c14304cb3a269a39f6e0770ce3e (*: use new Copyright (c) Balabit convention)
Branches: master and many more (405)
Follows: syslog-ng-3.7.0alpha1
Precedes: syslog-ng-3.7.0alpha2

    scl/scl.conf: use a wildcard include instead of referencing files one-by-one
    
    This makes it easier to work with installations that selectively installs
    various SCL snippets, and also makes the file less redundant.
    
    Signed-off-by: Balazs Scheidler <balazs.scheidler@balabit.com>
```

So, this can finally go away :)